update.sh

```diff
@@ -92,6 +92,10 @@ RUN rm -rf /var/lib/apt/lists/*
 # enable the universe
 RUN sed -i 's/^#\s*\(deb.*universe\)$/\1/g' /etc/apt/sources.list
 
+# make systemd-detect-virt return "docker"
+# See: https://github.com/systemd/systemd/blob/aa0c34279ee40bce2f9681b496922dedbadfca19/src/basic/virt.c#L434
+echo 'docker' > /run/systemd/container
```

Comment:
This needs `RUN` in front of it or we'll get `ECHO: unknown instruction` from Docker. :smile: